### ISSUE TITLE
Dialog Modal tabbing issue in iOS 14.2 with VoiceOver on

### ISSUE THREAD

**Issue by richiegarcia** [w3c/aria-practices#1626]
https://www.w3.org/TR/wai-aria-practices-1.1/examples/dialog-modal/dialog.html
In Safari iOS 14.2 with VoiceOver enabled, tabbing  in modal does not work properly. The focus gets trapped on the "Add Delivery Address" behind the modal. The user can not tab back or forward.

**Comment by marcus-herrmann** [comment]
As far as I know, the reason is that this example works with `aria-modal` and iOS VoiceOver still doesn't support it:

https://a11ysupport.io/tech/aria/aria-modal_attribute

**Comment by carmacleod** [comment]
I believe it may work (a bit) better in Safari if you use visibility: hidden instead of display: none.
